conf/reference.conf

```diff
@@ -1,4 +1,11 @@
 org.zalando.markscheider {
+
+  durationUnit = "SECONDS"
```

Comment:
here as well, please do not change it to seconds. Change in your service through configuration.